api/Document.json

```
@@ -3523,6 +3523,89 @@
               "deprecated": true
             }
           }
+        },
+        "insertHTML": {
+          "__compat": {
+            "description": "`insertHTML` command",
+            "support": {
+              "chrome": {
+                "version_added": "1"
+              },
+              "chrome_android": "mirror",
+              "edge": {
+                "version_added": "12"
+              },
+              "firefox": [
+                {
+                  "version_added": "69",
+                  "notes": [
+                    "From Firefox 82, nested calls are not supported (return `false`). See [bug 1634262](https://bugzil.la/1634262).",
+                    "Before Firefox 89, manipulating the content of `<input>` and `<textarea>` elements using `Document.execCommand()` commands requires workarounds (see [bug 1220696](https://bugzil.la/1220696))."
+                  ]
+                },
+                {
+                  "version_added": "1",
+                  "version_removed": "69",
+                  "partial_implementation": true,
+                  "notes": "Only supported for [`HTMLDocument`](https://developer.mozilla.org/docs/Web/API/HTMLDocument), not all `Document` objects."
+                }
+              ],
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": "4"
+              },
+              "oculus": "mirror",
+              "opera": {
+                "version_added": "9"
+              },
+              "opera_android": {
+                "version_added": "10.1"
+              },
+              "safari": {
+                "version_added": "1.3"
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror",
+              "webview_ios": "mirror"
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": false,
+              "deprecated": true
```

Isn't this standard track? (And if yes, what spec URL could we set?)

```suggestion
              "standard_track": true,
              "deprecated": false
```

---

This is nested in the parent `execCommand()`. That's deprecated according to BCD, so doesn't that means this is deprecated right?

But yes, it is standard track.

```suggestion
              "standard_track": true,
              "deprecated": true
```